### PR TITLE
feat(badges): adding badges and test coverage

### PR DESCRIPTION
**Pull request by milos1290**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+![Leanplum - ](Leanplum.svg)
+
+<p align="center">
+<a href="https://nodei.co/npm/leanplum-sdk/"><img src="https://nodei.co/npm/leanplum-sdk.png?downloads=true&downloadRank=true&stars=true" alt="CocoaPods compatible" /></a>
```
Comment (alexisoyama) [inline README.md]: Remove Cocoapods

File: README.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+</p>
+<p align="center">
+<img src='https://jenkins.leanplum.com/buildStatus/icon?job=javascript-sdk' alt="Build status">
```
Comment (alexisoyama) [inline README.md]: Should we have develop and master like for other sdks?

File: README.md (hunk 1 above)
Comment (milos1290) [reply README.md]: ah copy/paste :)

**Comment by milos1290** [comment]
test this please

**Comment by alexisoyama** [comment]
can you update it?

**Comment by alexisoyama** [comment]
The commits looks good. Can you make sure both builds on Jenkins are running? Right now it is empty. https://jenkins.leanplum.com/job/javascript-sdk-master/

**Comment by alexisoyama** [comment]
https://jenkins.leanplum.com/job/javascript-sdk-develop/

**Pull request merged**